classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] Here, only `:label` should be in `attr_accessor`. Kindly move `:students` to the `attr_reader`.